pkg/crypto/crypto.go

```diff
@@ -115,7 +115,7 @@ func DeviceEncryptionStatus(devicePath string) (mappedDevice, mapper string, err
 	if len(lines) < 1 {
 		return "", "", fmt.Errorf("device encryption status returned no stdout for %s", devicePath)
 	}
-	if !strings.HasSuffix(lines[0], " is active.") {
+	if !strings.Contains(lines[0], " is active") {
```

Comment:
Does it mean the original impl is problematic and malfunctional?

```suggestion
	if !strings.Contains(lines[0], " is active") {
```

Reply:
Suggest having a comment about the different messages.